File: CS/T299944/Models/FormLayoutHelper.cs
using DevExpress.Web;
using DevExpress.Web.Mvc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI.WebControls;

namespace T299944.Models
{
    public class FormLayoutHelper
    {
        static Action<MVCxFormLayoutItem> formLayoutItemSettingsMethod;
        public static Action<MVCxFormLayoutItem> FormLayoutItemSettingsMethod
        {
            get
            {
                if (formLayoutItemSettingsMethod == null)
                    formLayoutItemSettingsMethod = CreateFormLayoutItemSettingsMethod();
                return formLayoutItemSettingsMethod;
            }
        }
        static Action<MVCxFormLayoutItem> CreateFormLayoutItemSettingsMethod()
        {
            return itemSettings =>
            {
                dynamic editorSettings = itemSettings.NestedExtensionSettings;
                editorSettings.Properties.ValidationSettings.Display = Display.Dynamic;
                editorSettings.Properties.ValidationSettings.ErrorDisplayMode = ErrorDisplayMode.ImageWithText;
                editorSettings.Properties.ValidationSettings.ErrorTextPosition = ErrorTextPosition.Bottom;
                editorSettings.ShowModelErrors = true;
                editorSettings.Width = Unit.Pixel(200);
            };
        }
    }
}

File: VB/T299944/Models/FormLayoutHelper.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Web
Imports DevExpress.Web.Mvc
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI.WebControls

Namespace T299944.Models
	Public Class FormLayoutHelper
		Private Shared formLayoutItemSettingsMethod_Renamed As Action(Of MVCxFormLayoutItem)
		Public Shared ReadOnly Property FormLayoutItemSettingsMethod() As Action(Of MVCxFormLayoutItem)
			Get
				If formLayoutItemSettingsMethod_Renamed Is Nothing Then
					formLayoutItemSettingsMethod_Renamed = CreateFormLayoutItemSettingsMethod()
				End If
				Return formLayoutItemSettingsMethod_Renamed
			End Get
		End Property
		Private Shared Function CreateFormLayoutItemSettingsMethod() As Action(Of MVCxFormLayoutItem)
			Return Function(itemSettings) AnonymousMethod1(itemSettings)
		End Function
		
		Private Shared Function AnonymousMethod1(ByVal itemSettings As Object) As Boolean

            Dim editorSettings As Object = itemSettings.NestedExtensionSettings
			editorSettings.Properties.ValidationSettings.Display = Display.Dynamic
			editorSettings.Properties.ValidationSettings.ErrorDisplayMode = ErrorDisplayMode.ImageWithText
			editorSettings.Properties.ValidationSettings.ErrorTextPosition = ErrorTextPosition.Bottom
			editorSettings.ShowModelErrors = True
			editorSettings.Width = Unit.Pixel(200)
			Return True
		End Function
	End Class
End Namespace